tests/ui/never_type/exhaustive_patterns.stderr

```diff
@@ -14,6 +14,7 @@ LL | enum Either<A, B> {
 LL |     A(A),
 LL |     B(inner::Wrapper<B>),
    |     - not covered
+   = note: pattern `Either::B(_)` is currently uninhabited, but this variant contains private fields which may change in the future
```

Comment:
Could you add the appropriate magic comments to the corresponding rust file so we don't accidentally regress this?